src/bootstrap/workarounds.rs

```diff
@@ -0,0 +1,112 @@
+use bindings::{

```

Comment:
I'd like to avoid supporting these helpers unless there's an urgent blocking issue they resolve.

Reply:
I tried to signal that via the `workarounds` namespace. Want to drop them altogether or rename? Every app developer will need these helpers due to gaps in WinUI.

Reply:
Then perhaps just a `windows_app::window_handle(windows::IInspectable) -> isize` interop function and let developers do the rest by calling whatever APIs they need. Hopefully WinUI fills their gaps but at least then developers can plug whatever else they need in the meantime without us having to add wrappers for everything.